custom_model_runner/datarobot_drum/drum/gpu_predictors/utils.py

```diff
@@ -107,7 +107,7 @@ def _configure_ngc_client(self):
 class S3Client:
     def __init__(self, s3_url, credential):
         import boto3
-        from mlpiper.extra.aws_helper import AwsHelper
+        from aws_helper import AwsHelper
```

Comment:
I think this import is wrong. It looks like you moved it into GPU predictor folder so you can use a relative import here.